build.sbt

```diff
@@ -67,7 +67,7 @@ lazy val dockerSettings = Seq(
   dockerBaseImage := "eclipse-temurin:21.0.2_13-jdk-alpine",
   dockerRepository := Some("hmda"),
   dockerCommands := dockerCommands.value.flatMap {
-    case cmd@Cmd("FROM",_) => List(cmd, Cmd("RUN", "apk add --no-cache openssl"),
```

Comment:
Remove here since it isn't necessary, the upgrade resolves this cve, although we could use this base image as well: `eclipse-temurin:21.0.5_11-jdk-alpine`